crates/fuel-core/src/query/subscriptions/test.rs

```diff
@@ -100,11 +121,19 @@ enum TxStatus {
 enum FinalTxStatus {
     /// The transaction was successfully included in a block.
     Success,
+    /// The transaction has been executed, but the full block has not been produced yet.
+    SuccessDuringBlockProduction,
     /// The transaction was squeezed out of the txpool (or block)
     /// because it was not valid to include in the block.
     Squeezed,
+    /// Transaction was eligible for execution and inclusion in the block but
+    /// it was squeezed out by block producer.
+    SqueezedOutDuringBlockProduction,
     /// The transaction failed to execute and was included in a block.
     Failed,
+    /// Transaction was eligible for execution and inclusion in the block but
+    /// it failed during the execution.
+    FailureDuringBlockProduction,
```

Comment:
IT would be nice to have parity on `Failed` and `Failure`

Reply:
Updated in https://github.com/FuelLabs/fuel-core/pull/2752/commits/d702ca7e8254d75bc8d9825a74cbda64cfeb4bd5